src/styles/_subscription.scss

```diff
@@ -0,0 +1,52 @@
+.subscription {
```

Comment:
only files in the utils folder should start with the underscore `_` - fix it
that also might be the reason your style doesn't work - the problem with the path to the CSS file

if you need help to fix it - ask in the chat instead of GitHub - that's a much faster way to communicate